spinalcordtoolbox/scripts/sct_compute_mtsat.py

```diff
@@ -28,63 +28,68 @@ def get_parser():
                     'MRI. Magn Reson Med 2008;60(6):1396-1407.'
     )
 
-    mandatory = parser.add_argument_group("MANDATORY ARGUMENTS")
+    mandatory = parser.mandatory_arggroup
     mandatory.add_argument(
         "-mt",
-        required=True,
         help="Image with MT_ON",
         metavar=Metavar.file,
     )
     mandatory.add_argument(
         "-pd",
-        required=True,
         help="Image PD weighted (typically, the MT_OFF)",
         metavar=Metavar.file,
     )
     mandatory.add_argument(
         "-t1",
-        required=True,
         help="Image T1-weighted",
         metavar=Metavar.file,
     )
 
-    optional = parser.add_argument_group('OPTIONAL ARGUMENTS')
-    optional.add_argument(
+    sidecar_overrides = parser.add_argument_group(
+        "JSON SIDECAR OVERRIDES",
+        description="Any arguments within this group will override their corresponding values in the JSON side-car files "
+                    "of the MRI sequences you have provided. Otherwise, all values will be fetched from the "
+                    "associated JSON sidecar, assuming it exists. If not, you should provide the corresponding metrics "
+                    " using these arguments."
+    )
+    sidecar_overrides.add_argument(
```

Comment:
Another very sensible change! Thank you for de-duplicating the old message (and providing a very clear representation of the behavior + user guidance).